2024/ig-exploration.html

```diff
@@ -147,25 +147,34 @@ <h1 id="title"><i class="todo">DRAFT</i> Exploration Interest Group Charter</h1>
 
       <div id="background" class="background">
         <h2>Motivation and Background</h2>
-
         <p>
-          The goal of Exploration Interest Group provides a platform to help W3C community
+          The goal of Exploration Interest Group provides a platform to help the W3C community
           investigating emerging technology trends, analyzing their impacts on the evolution
           of Web technologies, and proposing ways for W3C to collaborate shaping the trends
           for the benefit of the Web users. The ways include organizing Workshops, publishing
           Reports and creating topic oriented Task Forces etc., assisting the Web community
           seeking innovative responses to continually strengthen the innovation of W3C and the
           creativity of Web technologies.
         </p>
+        <p>
+          Examples of who people may take advantage of this forum include:
+        </p>
+        <ul>
+          <li>the
+          <a href='https://www.w3.org/policies/process/#team'>W3C Team</a> to refine
+          charter drafts;</li>
+          <li>the <a href="https://www.w3.org/community/groups/">community groups</a> to
+          receive feedback on their potential next steps.</li>
+        </ul>
       </div>
 
       <section id="scope" class="scope">
         <h2>Scope</h2>
         <ul>
 <li>Monitor industry and technology trends that might impact the web;</li>
-<li>Monitor responses from the industries, SDOs (including W3C CGs, TPAC Breakout sessions, Workshops, Liaison Reports), and Open Source communities on those trends. Work with W3C Team to invite W3C Liaisons to SDOs or other insightful SDOs experts, reporting back on potential areas of investigations;</li>
-<li>Prioritize the trends and propose organizing Workshops, publishing Reports or creating topic oriented Task Forces, for further elaboration;</li>
-<li>Propose chartering potential topics to the Strategy Funnel at the stage of incubation;</li>
+<li>Monitor responses from the industries, SDOs (including W3C CGs, TPAC Breakout sessions, <a href='https://www.w3.org/events/workshops/'>Workshops</a>, <a href='https://www.w3.org/liaisons/'>Liaison Reports</a>), and Open Source communities on those trends. Work with W3C Team to invite W3C Liaisons to SDOs or other insightful SDOs experts, reporting back on potential areas of investigations;</li>
+<li>Prioritize the trends and propose to the W3C Team <a href='https://github.com/w3c/strategy/labels/workshop'>organizing Workshops</a>, publishing Reports or creating topic oriented Task Forces, for further elaboration;</li>
```

Comment:
I'm not sure of adding "to the W3C Team" here. I think that's implied for Workshops, so not necessary to mention; but publishing Reports or creating Task Forces of this IG doesn't seem to need to be proposed "to the Team" -- the IG can just do it.

Reply:
I think there's no need to mention creation of task forces in the charter. The IG can organize its work however it wants.